Application/EdFi.Ods.SandboxAdmin/appsettings.json

```diff
@@ -10,18 +16,22 @@
   "UnobtrusiveJavaScriptEnabled": "true",
   "DefaultApplicationName": "Default Sandbox Application",
   "DefaultClaimSetName": "Ed-Fi Sandbox",
-  "DefaultOperationalContextUri": "uri://ed-fi-api-host.org",
-  "OAuthUrl": "http://localhost:54746/oauth/",
+  "DefaultOperationalContextUri": "",
+  "OAuthUrl": "",
   "SandboxAdminSQLCommandTimeout": "120",
-  "MailSettings": {
-    "Smtp": {
-      "UserName": "Bingo",
-      "Password": "Tingo",
-      "DeliveryMethod": "SpecifiedPickupDirectory",
-      "From": "noreply@ed-fi.org",
-      "SpecifiedPickupDirectory": {
-        "PickupDirectoryLocation": "C:\\Temp\\AdminConsole\\Artifacts\\Emails"
+  "ApiSettings": {
+    "Engine": ""
+  },
+  "User": {},
+    "MailSettings": {
+      "Smtp": {
+        "UserName": "Bingo",
+        "Password": "Tingo",
+        "DeliveryMethod": "SpecifiedPickupDirectory",
+        "From": "noreply@ed-fi.org",
+        "SpecifiedPickupDirectory": {
+          "PickupDirectoryLocation": "C:\\Temp\\AdminConsole\\Artifacts\\Emails"
```

Comment:
should the mail settings be set to empty and updated in the appsettings.user.json?